prow/github/helpers.go

```diff
@@ -34,7 +34,7 @@ const ImageSizeLimit = 5242880
 // HasLabel checks if label is in the label set "issueLabels".
 func HasLabel(label string, issueLabels []Label) bool {
 	for _, l := range issueLabels {
-		if strings.ToLower(l.Name) == strings.ToLower(label) {
+		if strings.EqualFold(l.Name, label) {
```

Comment:
does this more closely match GitHub's equality than what was there before?

Reply:
https://golang.org/pkg/strings/#EqualFold
```
EqualFold reports whether s and t, interpreted as UTF-8 strings, are equal under Unicode case-folding, which is a more general form of case-insensitivity.
```

You would perhaps never hit any issues as the labels you are using are in English. Use of strings.EqualFold is often encouraged as a best practice.